testing/mongo-cxx-driver/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname="mongo-cxx-driver"
+pkgver="3.4.0"
+_commit="131fa1a67acd45c0eebcbdcfee42b212af8d2e80" # master compiles with GCC8.3/CXX17
+pkgrel=0
+pkgdesc="C++ driver for MongoDB"
+url="http://mongocxx.org/"
+arch="all"
+license="Apache-2.0"
+makedepends="cmake mongo-c-driver-dev libbson-dev openssl-dev python3 snappy-dev"
+subpackages="$pkgname-dev libbsoncxx"
+source="mongo-cxx-driver-$pkgver.tar.gz::https://github.com/mongodb/mongo-cxx-driver/archive/$_commit.tar.gz
+	01-dont-build-mongo-tests.patch"
+builddir="$srcdir/mongo-cxx-driver-$_commit"
+
+prepare() {
+	mkdir -p "$builddir/build"
+	default_prepare
+}
+
+build() {
+	cd "$builddir/build"
+	if [ "$CBUILD" != "$CHOST" ]; then
+		CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
+	fi
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		-DCMAKE_CXX_STANDARD=17 \
+		-DBSONCXX_POLY_USE_STD=ON \
+		${CMAKE_CROSSOPTS} ..
+	make
+}
+
+check() {
+	# mongo tests require a running server, so they are disabled.
+	# libbsoncxx tests are still ran
+	cd "$builddir/build"
+	make test
+}
+
+package() {
+	cd "$builddir/build"
+	make DESTDIR="$pkgdir" install
+	rm -rf "$pkgdir"/usr/share/mongo-cxx-driver/
+}
+
+libbsoncxx() {
```

Comment:
doesn't it also have its own -dev contents ?

Reply:
It does. I see you changed mongo-c-driver. I will modify to suit.

Reply:
:tada: working now.